Sources/Plasma/NucleusLib/pnMessage/plMessage.cpp

```diff
@@ -344,13 +333,10 @@ int plMsgCStringHelper::Peek(char *& str, hsStream* stream, const uint32_t peekO
         {
             str = new char[strlen+1];
             str[strlen] = '\0';
-            if (strlen) {
-                stream->LogRead(strlen,str,"CString");
-                stream->LogStringString(ST::format("Value: {}", str).c_str());
-            }
+            if (strlen)
+                stream->Read(strlen, str);
```

Comment:
While we're here, I think we should move the null-terimation to be done post-read instead of pre-read. Just in case someone is sending us very gnawty strings....

Reply:
It actually doesn't matter, since we're not reading into that final element...  But yeah, it could be moved for clarity.